modules/integration/src/test/scala/scala/cli/integration/BloopTests.scala

```diff
@@ -31,9 +28,9 @@ class BloopTests extends munit.FunSuite {
     shouldRestart: Boolean
   ): Unit = TestUtil.retryOnCi() {
     dummyInputs.fromRoot { root =>
+      BloopUtil.killBloop()
 
       val bloop = BloopUtil.bloop(currentBloopVersion, bloopDaemonDir)
-      bloop("exit").call(cwd = root, stdout = os.Inherit, check = exitCheck)
```

Comment:
bloop("exit") still does not work, replaced with `BloopUtil.killBloop` cc @alexarchambault